chore: added emojis, fixed typos and grammatical errors and spaced text out

HareerSakhi opened the pull request:
Please check my changes. Thanks (:

ming-tsai reviewed (changes requested):
Hi @HareerSakhi,
Please do not change the text to the title, only add the emoji or color

HareerSakhi commented:
I have removed all of the text size changes. Thanks

ming-tsai commented on english/normal-dungeon/4/4.md:
```diff
@@ -1,5 +1,5 @@
-You open the door to your neighbour's house, and the first thing you see is a trail of blood💉 streaked on the floor...
+ You open the door 🚪 to your neighbour's house 🏡, and the first thing you see is a <span style= "color:red">trail of blood💉 streaked on the floor...</span>
```
DO NOT use HTML code on the markdown file. if you are more advanced and know ReactJs you could beautify the page

HareerSakhi commented:
I have gotten rid of the inline HTML, and made the page more visible by spacing out the text a little bit, added more emojis, and fixed some typos and grammar errors that I spotted. Thanks

HareerSakhi commented:
I have gotten rid of the inline HTML, and made the page more visible by spacing out the text a little bit to make the writing more visible, added more emojis, and fixed some typos and grammar errors that I spotted. Thanks.

HareerSakhi commented:
I have also completed folder 5 of the english normal-dungeon, adding more emojis (and also adding spaces before them), fixing grammatical errors, fixed one or two of the sentences so that they make more sense, and spacing text out.

ming-tsai reviewed (approved):
Hi @HareerSakhi,
Nice work, and welcome to be part of the team.
> The invitation [link](https://github.com/MakeContributions)
